beta/project-griffon/set-up-project-griffon/README.md

```diff
@@ -254,5 +285,27 @@ ACPGriffon.startSession(sessionUrl, function(handleCallback) {
 });
 ```
 {% endtab %}
+
+{% tab title="Unity" %}
+
+### startSession
+
+#### C#
+
+#### Syntax
+
+```csharp
+ACPGriffon.StartSession(url);
```

Comment:
This syntax should match the format seen at https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/pull/308/files#diff-3cb62446150e11a29b0f96634b637df3R54

Reply:
done